lib/installation.js

```diff
@@ -65,6 +68,23 @@ async function installDependencies () {
   installationLogger.log('... Done')
 }
 
+function sleep (ms) {
+  return new Promise((resolve) => { setTimeout(resolve, ms) })
+}
+
+async function checkGIT () {
+  try {
+    execSync('git --version')
+  } catch ({ message }) {
+    installationLogger.log('Git is not installed.')
+    await app.displayNotification(
+      'Before you can install Form Builder you will first need to install Git.', { phase: 'Install Editor' }
```

Comment:
do we want to provide a link to instructions somewhere on how to do this? i suppose that opens up difficulty in finding some which covers several platforms . . .

Reply:
Maybe adding to the runbook guides and link here?

Reply:
We can have a guide that cover Mac for now.

Reply:
yeah i think that would be cool 👍

Reply:
Done! I will create shortly the installation instruction on the guide (the same link that it is committed here).